collectors/python.d.plugin/uwsgi/README.md

```diff
@@ -36,6 +32,6 @@ localhost:
   port     : 1717
 ```
 
-When no configuration file is found, module tries to connect to TCP/IP socket: `localhost:1717`.
+If Netdata can't find a configuration file, the module tries to connect to thde TCP/IP socket at `localhost:1717`.
```

Comment:
> thde

a typo

Reply:
> If Netdata can't find a configuration file

> collectors/python.d.plugin/uwsgi/README.md

Technically it is not Netdata. `python.d.plugin` (external plugin) finds config files.